logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationMapper.kt

```diff
@@ -96,6 +109,152 @@ internal class ConversationMapperImpl(
+                        mutedStatus = conversationStatusMapper.fromMutedStatusDaoModel(mutedStatus),
+                        removedBy = null, // todo: ask how to calculate?
+                        creatorId = PlainId(""), // deprecated
+                        lastNotificationDate,
+                        lastModifiedDate,
+                        lastReadDate,
+                        access = accessList.map { it.toDAO() },
+                        accessRole = accessRoleList.map { it.toDAO() },
+                        isSelfUserMember = isMember == 1L,
+                        isCreator = isCreator == 1L,
+                    )
+                )
+            }
+
+            ConversationEntity.Type.ONE_ON_ONE -> {
+                ConversationDetails.OneOne(
+                    conversation = Conversation(
+                        id = idMapper.fromDaoModel(id),
+                        name = name,
+                        type = type.fromDaoModelToType(),
+                        teamId = teamId?.let { TeamId(it) },
+                        protocol = protocolInfoMapper.fromEntity(protocolInfo),
+                        mutedStatus = conversationStatusMapper.fromMutedStatusDaoModel(mutedStatus),
+                        removedBy = null, // todo: ask how to calculate?
+                        creatorId = PlainId(""), // todo: deprecated
+                        lastNotificationDate,
+                        lastModifiedDate,
+                        lastReadDate,
+                        access = accessList.map { it.toDAO() },
+                        accessRole = accessRoleList.map { it.toDAO() },
+                        isSelfUserMember = isMember == 1L,
+                        isCreator = isCreator == 1L,
+                    ),
+                    otherUser = OtherUser(
+                        id = idMapper.fromDaoModel(otherUserId.requireField("otherUserID in OneOnOne")),
+                        name = name,
+                        accentId = 0,
+                        userType = domainUserTypeMapper.fromUserTypeEntity(userType),
+                        availabilityStatus = userAvailabilityStatusMapper.fromDaoAvailabilityStatusToModel(userAvailabilityStatus),
+                        deleted = type != ConversationEntity.Type.GROUP && otherUserId != null,
+                        botService = botService?.let { BotService(it.id, it.provider) },
+                        handle = null,
+                        completePicture = previewAssetId?.let { idMapper.fromDaoModel(it) },
+                        previewPicture = previewAssetId?.let { idMapper.fromDaoModel(it) },
+                        teamId = teamId?.let { TeamId(it) }
+                    ),
+                    connectionState = connectionStatusMapper.fromDaoModel(connectionStatus),
+                    legalHoldStatus = LegalHoldStatus.DISABLED,
+                    userType = domainUserTypeMapper.fromUserTypeEntity(userType),
```

Comment:
Would this be the self user `userType` or the other user `userType` ?

Reply:
it's the other thing! like federated and...